### PR TITLE
New Thrasher – Keep It In The Ground

### PR DESCRIPTION
**Pull request by sammorrisdesign**
screen shot 2015-03-06 at 11 56 57

Currently first on: http://m.code.dev-theguardian.com/thrashers-live

### CODE REVIEW

File: embeds/keep-it-in-the-ground/hashmap.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "d2001_100_connection_ref.jpg": "d2001_100_connection_ref.a654072c.jpg",
+  "gormley.jpg": "gormley.a654072c.jpg",
+  "d2003_024_evening_ref.jpg": "d2003_024_evening_ref.e7fc7fab.jpg",
+  "gormley--yellow.jpg": "gormley--yellow.17e880fa.jpg",
```
Comment (phamann) [inline embeds/keep-it-in-the-ground/hashmap.json]: What are all these other images, I don't see them referenced anywhere?

File: embeds/keep-it-in-the-ground/hashmap.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "gormley.png": "gormley.817e28ee.png"
```
Comment (phamann) [inline embeds/keep-it-in-the-ground/hashmap.json]: I ran this through ImageOptim and got another 7% savings...

File: embeds/keep-it-in-the-ground/style.scss [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+    @include mq(tablet) {
+        position: absolute;
+        bottom: 12px;
+    }
+
+    br {
+        @include mq($until: tablet) {
+            // display: none;
+        }
```
Comment (phamann) [inline embeds/keep-it-in-the-ground/style.scss]: Whats this doing?

File: embeds/keep-it-in-the-ground/style.scss [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+    }
+
+    @include mq(desktop) {
+        padding-right: 26%;
+    }
+}
+
+.keep-it-in-the-ground__attribution {
+    display: block;
+    color: mix($c-text, $c-background, 80%);
```
Comment (phamann) [inline embeds/keep-it-in-the-ground/style.scss]: :+1:

**Comment by phamann** [comment]
:+1: Looks good to me other than the image comments.

File: embeds/keep-it-in-the-ground/hashmap.json (hunk 2 above)
Comment (sammorrisdesign) [inline embeds/keep-it-in-the-ground/hashmap.json]: [Compressor](http://compressor.io) has let me down...

**Comment by sndrs** [comment]
looks like this in IE9, 1280x800, on live network front...

screen shot 2015-03-06 at 12 43 17

**Comment by sammorrisdesign** [comment]
@sndrs Ah, that would be the lack of animation... just updated to not fade in.

**Pull request merged**